apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -135,6 +135,13 @@ export interface IExtractorConfigPrepareOptions {
    * `@microsoft/api-extractor/extends/tsdoc-base.json`.
    */
   tsdocConfigFile?: TSDocConfigFile;
+
+  /**
+   * When preparing the configuration object, folder and file paths referenced in the configuration are checked
+   * for existence, and an error is reported if they are not found.  This option can be used to disable this
+   * check. This may be useful when preparing a configuration file for an un-built project.
+   */
+  ignoreMissingConfigTargets?: boolean;
```

Comment:
Is it possible for a user to set this via config file (i.e. `IConfigFile`)? It doesn't look like it's possible. Should it be?

Reply:
I don't think it makes sense to allow a config file to set this. It's somewhat of a `loadWithoutValidation` tweak, and having that value in the config file feels odd. By the time API Extractor is invoked, all the required files should be available. This change is more to allow loading and inspecting the configuration without invoking API Extractor, so it makes sense to keep it within the API interface IMO